stl/inc/chrono

```diff
@@ -1331,15 +1331,16 @@ namespace chrono {
         return *this;
     }
 
-    inline constexpr day _Last_day_table[] = {
-        day{31}, day{28}, day{31}, day{30}, day{31}, day{30}, day{31}, day{31}, day{30}, day{31}, day{30}, day{31}};
+    // To prevent UB by going out of bounds, four extra days with an invalid day are added.
+    inline constexpr day _Last_day_table[] = {day{31}, day{28}, day{31}, day{30}, day{31}, day{30}, day{31}, day{31},
+        day{30}, day{31}, day{30}, day{31}, day{42}, day{42}, day{42}, day{42}};
 
     _NODISCARD constexpr day _Last_day(const year& _Year, const month& _Month) {
         if (_Month == month{2} && _Year.is_leap()) {
             return day{29};
         }
 
-        return _Last_day_table[static_cast<unsigned int>(_Month) - 1];
+        return _Last_day_table[(static_cast<unsigned int>(_Month) - 1) & 15u];

```

Comment:
Style: I personally find it slightly jarring to see decimal numbers in a bitwise context (for some reason, `15` doesn't immediately read as `1` less than `16`). I would recommend any of:
* `& 0xF`
* `& 0b1111` (probably too verbose, so less preferred)
* `% 16` (and trust the codegen will be good; this would *work* for any decimal value, it just happens be efficient for powers of 2, which are easy to recognize in decimal)